sys/include/checksum.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_checksum Checksum
+ * @ingroup     sys
+ * @brief       Checksum function libraries
+ */
+
+#ifndef CHECKSUM_H
+#define CHECKSUM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
```

Comment:
Maybe import of all the checksum sub-headers would be nice, otherwise I prefer to have the doc in a separate `doc.txt` file.